data/language/en-GB.txt

```diff
@@ -2885,7 +2885,7 @@ STR_5755    :Incorrect Software Version (Server is using {STRING})
 STR_5756    :Bad Password
 STR_5757    :Server Full
 STR_5758    :{OUTLINE}{GREEN}{STRING} has joined the game
-STR_5759    :Downloading map … ({INT32} / {INT32}) KiB
+STR_5759    :Downloading map …
```

Comment:
This will need to be updated immediately for all languages after merging. This is fine, just making sure to mention it.

Reply:
Yes, I was planning on filing a PR to the Localisation repo for it right after.

Reply:
https://github.com/OpenRCT2/Localisation/pull/2842